vividus-tests/src/main/resources/story/integration/ActionSteps.story

```diff
@@ -3,10 +3,86 @@ Description: Integration tests for ActionSteps class.
 Meta:
     @epic vividus-plugin-web-app
 
+Scenario: Open page
+Given I am on a page with the URL 'https://vividus-test-site.herokuapp.com'
+
+Scenario: Action verification MOVE_BY_OFFSET
+When I go to the relative URL '/mouseEvents.html'
+Then number of elements found by `By.xpath(//*[@id='moveCount' and text()='0'])` is equal to `1`
+When I execute sequence of actions:
+|type          |argument                                   |
+|MOVE_TO       |By.xpath(//div[contains(., 'Mouse move!')])|
+|MOVE_BY_OFFSET|(0, 10)                                    |
+|MOVE_BY_OFFSET|(0, -10)                                   |
+|MOVE_BY_OFFSET|(0, 10)                                    |
+|MOVE_BY_OFFSET|(0, -10)                                   |
+Then number of elements found by `By.xpath(//*[@id='moveCount' and text()='5'])` is equal to `1`
+
 Scenario: Action verification MOVE_TO
-Given I am on a page with the URL 'https://www.w3schools.com/jsref/tryit.asp?filename=tryjsref_onmousemove_over_enter'
-When I switch to a frame with the attribute 'id'='iframeResult'
+When I go to the relative URL '/mouseEvents.html'
+Then number of elements found by `By.xpath(//*[@id='enterCount' and text()='0'])` is equal to `1`
+When I execute sequence of actions:
+|type   |argument                                    |
+|MOVE_TO|By.xpath(//div[contains(., 'Mouse enter!')])|
+Then number of elements found by `By.xpath(//*[@id='enterCount' and text()='1'])` is equal to `1`
+
+Scenario: Action verification CLICK
+When I go to the relative URL '/mouseEvents.html'
+When I initialize the scenario variable `expectedText` with value `Good day!`
+Then the text '${expectedText}' does not exist
+When I execute sequence of actions:
+|type |argument                             |
+|CLICK|By.xpath(//button[text()='Click me'])|
+Then the text '${expectedText}' exists
+
+Scenario: Action verification CLICK_AND_HOLD
+When I go to the relative URL '/mouseEvents.html'
+When I change context to an element by By.xpath(//p[contains(text(), 'Click and hold the text!')])
+Then the context element has the CSS property 'color'='rgba(0, 0, 0, 1)'
 When I execute sequence of actions:
-|type   |argument                                                   |
-|MOVE_TO|By.xpath(//div[contains(., 'onmouseover: Mouse over me!')])|
-Then the text 'onmouseover: 1' exists
+|type          |argument         |
+|CLICK_AND_HOLD|By.xpath(self::*)|
+Then the context element has the CSS property 'color'='rgba(255, 0, 0, 1)'
+
+Scenario: Action verification RELEASE
+When I go to the relative URL '/mouseEvents.html'
+When I change context to an element by By.xpath(//p[contains(text(), 'Click and hold the text!')])
+Then the context element has the CSS property 'color'='rgba(0, 0, 0, 1)'
+When I execute sequence of actions:
+|type          |argument         |
+|CLICK_AND_HOLD|By.xpath(self::*)|
+|RELEASE       |By.xpath(self::*)|
+Then the context element has the CSS property 'color'='rgba(0, 0, 0, 1)'
```

Comment:
this test may produce false-positive results, since the state of application is not changed after the action